### ISSUE TITLE
First call to gensym takes about 5s on macOS

### ISSUE THREAD

**Issue by qzivli** [cisco/ChezScheme#351]
First call to `gensym` takes about 5s on macOS, and `time` does not handled it right.

e.g.,
```
(time (gensym))
    no collections
    0.000000000s elapsed cpu time
    0.000000000s elapsed real time
    48 bytes allocated
```
Actually, above takes about 5s. After the first call, `gensym` works well.

Chez Scheme version: 9.5.1
macOS version: Mojave

**Comment by weinholt** [comment]
This could be related to #338 if takes five seconds because your computer's hostname is invalid and the DNS query to resolve it times out.

The code that I suspect is here:
https://github.com/cisco/ChezScheme/blob/41eca9aeea5d036877d09f996414c1f9311ecdb8/c/stats.c#L77-L107

Does the five second delay go away if you add "return 0;" on line 82 and rebuild?

**Comment by qzivli** [comment]
Should be closed, see #338 and 2fd3db6.
